### DISCUSSION TITLE
Is it possible to `--force` a single task but not its dependencies?

### DISCUSSION THREAD

**Discussion by mattbsly** [vercel/turborepo#7101] [Help] [answered]
I'd like to be able to run something like `turbo run deploy --force=deploy`, where the deploy step would be forced to re-run but the dependencies, such as `build`, use the cache. Is this possible?

**Comment by mehulkar** [comment, marked as answer]
No, this is not possible, but you could do it in two steps:

```bash
turbo run deploy --only && turbo run build
```

You could also set `cache: false` in turbo.json for the `deploy` task if this is the way you always want to run it.

**Reply by mattbsly** [reply]
Thank you! That's exactly what I'm looking for.
